modules/querying/pages/operators-and-expressions.adoc

```diff
@@ -207,6 +207,23 @@ condition := expr
 Strings are compared based on standard lexicographical ordering: +
 (space) < (digit) < (uppercase_letter) < (lowercase_letter).
 
+=== Comparator Support
+Bitwise accumulators support the standard comparisonOperator := "<" | "< =" | ">" | ">=" | "==" | "!=".
+
+=== Example
+
+```
+WHERE (@@BitwiseAccum1 > 32) AND (@@BitwiseAccum2 < 1024)
+```
+
+The above condition filters results where @@BitwiseAccum1 is greater than 32 and @@BitwiseAccum2 is less than 1024.
+
+**Implementation Details**
+
+**Default-Length Bitwise Accumulators:** Automatically converted to integers for comparison.
+
+**Extended-Length Bitwise Accumulators (greater than 64 bits)**: Comparators are overridden, which may result in slower performance compared to default-length accumulators.
```

Comment:
I said last time that is it too much detail. 
Better:

NOTE:  Extended-Length bitwise accumulators (greater than 64 bits) may perform comparisons slower than default length accumulators